mne/bem.py

```diff
@@ -1953,13 +1890,8 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
 
     mri_dir = Path(mri_dir)
     bem_dir = Path(bem_dir)
-    if flash_path is None:
-        flash_path = mri_dir / 'flash' / 'parameter_maps'
-    else:
-        warn("The flash_path parameter is deprecated and will be removed "
-             "in version 1.1. Use the flash5_img parameter instead.")
-        flash_path = Path(flash_path).resolve()
     subjects_dir = env['SUBJECTS_DIR']
+    flash_path = (mri_dir / 'flash' / 'parameter_maps').resolve()
```

Comment:
I think the `resolve` will fail if `makedirs` hasn't been called yet, so you need
```suggestion
    flash_path = (mri_dir / 'flash' / 'parameter_maps').resolve(strict=False)
```
https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve

Reply:
`strict=False` is the default